### DISCUSSION TITLE
Nested directories don't seem to work when deploying on Vercel

### DISCUSSION THREAD

**Discussion by rankincodes** [resend/react-email#1242] [Q&A]
Steps I took to reproduce:

1. Download react email starter
2. deploy to vercel (works fine)
3. create a subdirectory and put an email in it
4. redeploy (internal server error on preview server)

the subdirectories work fine on the dev server but don't seem to work on the built version. Anyone else experience this?

**Comment by gabrielmfern** [comment]
Yup, we also had this with the demo and it should get fixed by #1203
